maven/bnd-maven-plugin/src/main/java/aQute/bnd/maven/plugin/BndMavenPlugin.java

```diff
@@ -119,6 +120,9 @@
 	@Parameter(property = "bnd.skip", defaultValue = "false")
 	private boolean									skip;
 
+	@Parameter(property = "bnd.packagingTypes", defaultValue = PACKAGING_JAR + "," + PACKAGING_WAR)
+	private String[]								packagingTypes;
```

Comment:
Can we not make this `List<String>` to avoid the need for `Arrays.asList` below?